src/ui/components/Popover.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {Icon, useTheme, Caption} from '@ui/library';
+import {Popable} from 'react-native-popable';
+import {View, ViewStyle} from 'react-native';
+
+export type PopableProps = {
+  content: string;
+  contentStyle?: ViewStyle;
```

Comment:
There is no need to add a styles prop, `content` should be of type `React.ReactNode` and the component can be used exactly how the `Popable` component is used here